db_release_scripts/mosip_regprc/sql/1.2_regprc-scripts_release.sql

```diff
@@ -13,13 +13,89 @@
+
+
+ALTER TABLE regprc.reg_manual_verification ADD COLUMN workflow_instance_id character varying(36);
+
+ALTER TABLE regprc.reg_lost_uin_det ADD COLUMN workflow_instance_id character varying(36);
+
+ALTER TABLE regprc.reg_bio_ref ADD COLUMN workflow_instance_id character varying(36);
+ALTER TABLE regprc.reg_bio_ref ADD COLUMN process character varying(36);
+ALTER TABLE regprc.reg_bio_ref ADD COLUMN iteration integer DEFAULT 1;
+
+
+UPDATE regprc.registration a SET workflow_instance_id = b.workflow_instance_id FROM regprc.registration_list b WHERE a.reg_id = b.reg_id;
+UPDATE regprc.individual_demographic_dedup a SET workflow_instance_id = b.workflow_instance_id FROM regprc.registration_list b WHERE a.reg_id = b.reg_id;
+UPDATE regprc.reg_manual_verification a SET workflow_instance_id = b.workflow_instance_id FROM regprc.registration_list b WHERE a.reg_id = b.reg_id;
+UPDATE regprc.reg_lost_uin_det a SET workflow_instance_id = b.workflow_instance_id FROM regprc.registration_list b WHERE a.reg_id = b.reg_id;
+UPDATE regprc.reg_bio_ref a SET workflow_instance_id = b.workflow_instance_id FROM regprc.registration_list b WHERE a.reg_id = b.reg_id;
+
+create index idx_rbioref_crdtimes on regprc.reg_bio_ref (cr_dtimes);
+
+DROP INDEX idx_idemogd_namedobgender;
+create index idx_idemogd_namedobgender on regprc.individual_demographic_dedup (name, dob,gender);
+
+create index idx_rbioref_crdtimes on regprc.reg_bio_ref (cr_dtimes);
+
+create index idx_rmanvrn_reqid on regprc.reg_manual_verification (request_id);
+
+create index idx_rgstrn_ltstrbcode_ltststscode on regprc.registration (latest_trn_dtimes, latest_trn_status_code);
+
+create index idx_rgstrnlst_pcktid on regprc.registration_list (packet_id);
+create index idx_rgstrnlst_aireqid on regprc.registration_list (additional_info_req_id);
+
+ALTER TABLE regprc.individual_demographic_dedup DROP CONSTRAINT pk_idemogd_id;
+ALTER TABLE regprc.individual_demographic_dedup ALTER COLUMN workflow_instance_id SET NOT NULL;
+ALTER TABLE regprc.individual_demographic_dedup ADD CONSTRAINT pk_idemogd_id PRIMARY KEY (workflow_instance_id,lang_code);
+
+ALTER TABLE regprc.reg_bio_ref DROP CONSTRAINT pk_regbref_id;
+ALTER TABLE regprc.reg_bio_ref ALTER COLUMN workflow_instance_id SET NOT NULL;
+ALTER TABLE regprc.reg_bio_ref ADD CONSTRAINT pk_regbref_id PRIMARY KEY (bio_ref_id,workflow_instance_id);
+
+ALTER TABLE regprc.reg_lost_uin_det DROP CONSTRAINT pk_rlostd;
+ALTER TABLE regprc.reg_lost_uin_det ALTER COLUMN workflow_instance_id SET NOT NULL;
+ALTER TABLE regprc.reg_lost_uin_det ADD CONSTRAINT pk_rlostd PRIMARY KEY (workflow_instance_id);
+
+ALTER TABLE regprc.reg_manual_verification DROP CONSTRAINT pk_rmnlver_id;
+ALTER TABLE regprc.reg_manual_verification ALTER COLUMN workflow_instance_id SET NOT NULL;
+ALTER TABLE regprc.reg_manual_verification ADD CONSTRAINT pk_rmnlver_id PRIMARY KEY (workflow_instance_id,matched_ref_id,matched_ref_type);
+
+ALTER TABLE regprc.registration DROP CONSTRAINT pk_reg_id;
```

Comment:
currently "pk_reg_id" is used as foreign key in below tables. So the CONSTRAINT cant be dropped without altering below tables. Please remove CONSTRAINT from below tables and add workflow_instance_id as foreign key
constraint fk_idemogd_reg on table individual_demographic_dedup depends on index pk_reg_id
constraint fk_rmnlver_reg on table reg_manual_verification depends on index pk_reg_id
constraint fk_regbrf_reg on table reg_bio_ref depends on index pk_reg_id
constraint fk_rlostd_reg on table reg_lost_uin_det depends on index pk_reg_id

Reply:
This has been done.